ui/src/app/icons/playground-mode-icon.tsx

```diff
@@ -1,30 +1,85 @@
 import * as React from 'react';
 
+import colors, {colorWithWhiteness} from 'app/styles/colors';
 
-export const PlaygroundModeIcon = () => <svg version='1.1'
```

Comment:
Wow, I didn't know we're embedding SVG within some of our code. Is there a way to avoid doing this? It feels so wrong... :)

Reply:
Maybe. I was noticing that it wouldn't override fill values without this, which is why I factored the playground icon this way. Not super happy with it though. I'm curious if Terra does something better.